### PR TITLE
fix the rule to read puppeteer configFile

### PR DESCRIPTION
**Pull request by i10416**
fix bug
- fix condition to read `puppeteer-config.json` and move a variable to local scope.
- add better log message for debug

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -25,16 +25,18 @@ const processBlock = (block) => {
   const inputFilePath = path.resolve(path.join('.', `${PREFIX}_${hash}.mmd`));
   const outputFilePath = path.resolve(path.join('.', `${PREFIX}_${hash}.${format}`));
   const configFilePath = path.resolve(path.join('.', `puppeteer-config.json`));
-  const configFileOption = !fs.existsSync(inputFilePath)
-    ? ''
-    : `-p ${configFilePath}`;
-  if (DEBUG) console.log(inputFilePath, outputFilePath);
+
   if (!fs.existsSync(inputFilePath)) { 
+    if (DEBUG) console.debug(`Input file not found. Writing ${inputFilePath}`);
```
Comment (i10416) [inline index.js]:
```suggestion
    if (DEBUG) console.log(`Input file not found. Writing ${inputFilePath}`);
```

File: index.js [hunk 2]
```diff
@@ -25,16 +25,18 @@ const processBlock = (block) => {
...
+
   if (!fs.existsSync(inputFilePath)) { 
+    if (DEBUG) console.log(`Input file not found. Writing ${inputFilePath}`);
     fs.writeFileSync(inputFilePath, source, { encoding: 'utf-8' });
   }
   if (!fs.existsSync(outputFilePath)) {
+    const configFileOption = fs.existsSync(configFilePath) ? `-p ${configFilePath}`  : '';
+    if (DEBUG) console.debug(
+      `Output file not found. Writing ${outputFilePath} with option: ${configFileOption}`
+    );
```
Comment (i10416) [inline index.js]:
```suggestion

  if (!fs.existsSync(inputFilePath)) { 
    if (DEBUG) console.log(`Input file not found. Writing ${inputFilePath}`);
    fs.writeFileSync(inputFilePath, source, { encoding: 'utf-8' });
  }
  if (!fs.existsSync(outputFilePath)) {
    const configFileOption = fs.existsSync(configFilePath) ? `-p ${configFilePath}`  : '';
    if (DEBUG) console.log(
      `Output file not found. Writing ${outputFilePath} with option: ${configFileOption}`
    );
```

File: index.js [hunk 3]
```diff
@@ -25,16 +25,18 @@ const processBlock = (block) => {
...
     fs.writeFileSync(inputFilePath, source, { encoding: 'utf-8' });
   }
   if (!fs.existsSync(outputFilePath)) {
+    const configFileOption = fs.existsSync(configFilePath) ? `-p ${configFilePath}`  : '';
+    if (DEBUG) console.log(
+      `Output file not found. Writing ${outputFilePath} with option: ${configFileOption}`
+    );
     const stdout = execSync(`npx mmdc ${configFileOption} -i ${inputFilePath} -o ${outputFilePath}`);
-    if (DEBUG) console.log(stdout);
+    if (DEBUG) console.debug(stdout);
```
Comment (i10416) [inline index.js]:
```suggestion
    if (DEBUG) console.log(stdout);
```

**Comment by i10416** [comment]
@TakuroFukamizu ping

**Comment by TakuroFukamizu** [comment]
@i10416 Thank you for your commit. your diff was merged by manually.

**Pull request closed without merging**